packages/json-kit/package.json

```diff
@@ -94,15 +94,15 @@
   "homepage": "https://github.com/breadboard-ai/breadboard#readme",
   "devDependencies": {
     "@ava/typescript": "^4.0.0",
-    "@google-labs/tsconfig": "^0.0.1",
-    "@types/node": "^18.16.3",
+    "@google-labs/tsconfig": "0.0.1",
+    "@types/node": "^20.11.5",
     "@typescript-eslint/eslint-plugin": "^6.20.0",
     "@typescript-eslint/parser": "^6.20.0",
     "ava": "^5.2.0",
-    "typescript": "^5.0.4"
+    "typescript": "^5.3.3"
   },
   "dependencies": {
-    "@google-labs/breadboard": "^0.10.0",
+    "@google-labs/breadboard": "0.10.0",
```

Comment:
I think we should pretty much never use `x.x.x` over `^x.x.x`. It means that only that exact version will be compatible, so it tends to cause duplication on install.

Reply:
sounds reasonable. added config to handle this